golem_messages/message/tasks.py

```diff
@@ -234,25 +234,52 @@ def __init__(self, *args, **kwargs):
 
 @library.register(TASK_MSG_BASE + 1)
 class WantToComputeTask(ConcentEnabled, base.Message):
+    """
+    A computation Offer.
+
+    A Provider sends it directly to a Requestor as a response to the Requestor's
+    Demand (Task) in order to get work (SubTask) to do.
+
+    Attributes:
+        node_name:          Provider's node name
+        perf_index:         Provider's Performance index
+        max_resource_size:  Provider's Storage size available for computation
+        max_memory_size:    Provider's RAM
+        price:              Offered price in GNT "WEI" (10e-18)
```

Comment:
I guess, we should define the unit here correctly ... I'm not completely sure now but I think that the price is expressed as "gnt wei / hour"

Reply:
well, I'd rather say that GM should be agnostic of such detail as payment scheme, isn't it?